docs/docker-integration/cloudformation.rst

```diff
@@ -0,0 +1,167 @@
+.. _cloudformation:
+
+.. raw:: html
+
+    <style>
+        .toctree-wrapper { display:none; }
+    </style>
+
+=========================================================
+Installing Flocker with Swarm on AWS using CloudFormation
+=========================================================
+
+The steps in this guide enable you to quickly deploy a Flocker/Swarm cluster, as illustrated below, which will create four EC2 instances on your AWS account:
+
+.. raw:: html
+
+	<div style="width:80%; margin-left:auto; margin-right:auto; margin-bottom:2em;">
+
+.. figure:: ../images/cloudformation.png
+    :alt: A diagram illustrating a cluster of four AWS EC2 instances running Flocker with Docker Swarm.
+
+.. raw:: html
+
+	</div>
+
+Follow the steps below to create your cluster.
+Once it's up and running we'll guide you through a tutorial to deploy a sample app.
+
+.. source material for this image: https://drive.google.com/open?id=0ByymF9bLBknGeXlPX1pTdXVZOGM
+
+.. raw:: html
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">Step 1</h2>
+			<p>Create an AWS Key Pair for secure login to your cluster.</p>
+		</div>
+		<div class="step-stages__step first">
+			<img src="../_images/01-keys-menu.png" alt="AWS key pairs section in console"/>
+            <span><a href="https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:sort=keyName" target="_blank">Log in to the AWS console</a>. This will open "N. Virginia" region, "Key Pairs" section.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/02-create-key.png" alt="Creating a new AWS key pair"/>
+            <span>Click "Create Key Pair". Give your key pair a meaningful name, like <strong>flocker-test</strong>. You'll need this later.</span>
```

Comment:
This screenshot and text are a little confusing - we seem to suggest user to create a Key Pair in ``N. Virginia`` region, irrespective of target region for his/her cluster.

It would be nice to have a sentence explaining that user must create Key Pair in the region where they intend to boot their cluster. For example, this is what Mesosphere says: ``Step 1: Select Your Region. This should be the same region where you will create your cluster.`` 

Thanks!